src/test/java/mil/dds/anet/test/resources/PersonResourceTest.java

```diff
@@ -562,6 +566,75 @@ public void testInsertCustomSensitiveInformation()
+    assertThat(orgs.getList().size()).isGreaterThan(0);
+    final Organization org = orgs.getList().stream()
+        .filter(o -> o.getShortName().equalsIgnoreCase("EF 6")).findFirst().get();
+    assertThat(org.getUuid()).isNotNull();
+
+    final PersonInput persInput =
+        PersonInput.builder().withRole(Role.ADVISOR).withName("Winner for Merging").build();
+    Person person = adminMutationExecutor.createPerson(FIELDS, persInput);
+    assertThat(person).isNotNull();
+    assertThat(person.getUuid()).isNotNull();
+    // Create a Position
+    final PositionInput testInput1 = PositionInput.builder().withType(PositionType.ADVISOR)
+        .withName("Test Position for person history edit  1")
+        .withOrganization(getOrganizationInput(org))
+        .withLocation(getLocationInput(getGeneralHospital())).withStatus(Status.ACTIVE).build();
+
+    final Position createdPos1 = adminMutationExecutor.createPosition(POSITION_FIELDS, testInput1);
+    assertThat(createdPos1).isNotNull();
+    assertThat(createdPos1.getUuid()).isNotNull();
+    assertThat(createdPos1.getName()).isEqualTo(testInput1.getName());
+
+    final PositionInput testInput2 = PositionInput.builder().withType(PositionType.ADVISOR)
+        .withName("Test Position for person history edit 2")
+        .withOrganization(getOrganizationInput(org))
+        .withLocation(getLocationInput(getGeneralHospital())).withStatus(Status.ACTIVE).build();
+
+    final Position createdPos2 = adminMutationExecutor.createPosition(POSITION_FIELDS, testInput2);
+    assertThat(createdPos2).isNotNull();
+    assertThat(createdPos2.getUuid()).isNotNull();
+    assertThat(createdPos2.getName()).isEqualTo(testInput2.getName());
+    PersonPositionHistory hist1 = new PersonPositionHistory();
+    Position p1 = new Position();
+    p1.setUuid(createdPos1.getUuid());
+    Position p2 = new Position();
+    p2.setUuid(createdPos2.getUuid());
+    hist1.setPosition(p1);
+    hist1.setStartTime(Instant.now().minus(100, ChronoUnit.DAYS));
+    hist1.setEndTime(Instant.now().minus(50, ChronoUnit.DAYS));
+    PersonPositionHistory hist2 = new PersonPositionHistory();
+    // hist2.setPerson(person);
+    hist2.setPosition(p2);
+    hist2.setStartTime(Instant.now().minus(49, ChronoUnit.DAYS));
+    hist2.setEndTime(Instant.now());
+    List<PersonPositionHistory> historyList = new ArrayList<>();
+    historyList.add(hist1);
+    historyList.add(hist2);
+    person.setPreviousPositions(historyList);
+    final PersonInput personInput = getPersonInput(person);
+    adminMutationExecutor.updatePersonHistory("", personInput);
+    final Person personUpdated =
+        adminQueryExecutor.person(PERSON_FIELDS_ONLY_HISTORY, personInput.getUuid());
+    // assertThat(personUpdated).isNotNull();
+    List<PersonPositionHistory> previousPositions = personUpdated.getPreviousPositions();
+    if (previousPositions == null) {
+      throw new Exception("nul geldi kardaş burayı düzelt");
```

Comment:
Throwing an exception is not a proper assertion. (And text should be English.)

Reply:
I wrote them for debugging for myself and  forget to remove